script.js

```diff
@@ -12,3 +12,387 @@ crossBtn.addEventListener('click', () => {
+</div>
+</section>`;
+}
+
+workInfo();
+
+const ul = document.querySelector('.work-ul');
+for (let i = 0; i < myProject.workTech.length; i += 1) {
+  const li = document.createElement('li');
+  li.textContent = myProject.workTech[i];
+  ul.appendChild(li);
+}
+
+/* popup section work button */
+
+const popUpContent = ` <div class="pupup-section-one">
+  <div class="popup-header">
+    <h2>Multi Post Stories</h2>
+    <a href="#"><img id="PopCross" src="images/popCloseBtn.png" class="popupClose" alt=""></a>
+  </div>
+ <div class="work-ul">
+  <ul>
+    <li>html</li>
+    <li>Bootstrap</li>
+    <li>Ruby on Rails</li>
+  </ul>
+ </div>
+ <div class="popup-img">
+ 
+  <img src="images/worklarge.png" alt="">
+ </div>
+  
+</div>
+<div class="pupup-section-two">
+  <p class="p-min">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining </p>
+  <p class="p-max">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent <br><br>
+    Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent
+  </p>
+  <button type="button" class="PoP">See Live</button>
+  <button type="button">See Source</button>
+</div>`;
+
+const body = document.querySelector('body');
+const workPopUp = document.querySelector('.pupup-window');
+
+const body2 = document.querySelector('.body');
+
+/* pop frist */
+
+const workBtn = document.querySelector('.work-see-project-btn');
+workBtn.addEventListener('click', () => {
+  body2.style.filter = 'blur(5px)';
+
+  workPopUp.innerHTML = popUpContent;
+
+  const popUpCrossBtn = document.querySelector('#PopCross');
+
+  let isClicked = true;
+
+  popUpCrossBtn.addEventListener('click', () => {
+    if (isClicked) {
+      workPopUp.style.display = 'none';
+      body.style.filter = 'blur(0)';
+
+      isClicked = false;
+    } else {
+      isClicked = true;
+    }
+
+    /* popup section see project button */
+  });
+});
+
+/* popC 1 */
+
+const workBtn1 = document.querySelector('.card-button');
+workBtn1.addEventListener('click', () => {
+  body2.style.filter = 'blur(5px)';
+
+  workPopUp.innerHTML = popUpContent;
```

Comment:
- [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md) for this project, "`When the user clicks (or taps) the button to check project details, the popup with details about the project appears. When the user clicks (or taps) the close (X) button, the popup disappears.`", when I click on the button to check project details, the screen just blurs out, I don't see the pop up, please see the image below and kindly fix that. Please make sure the popup with details about the project appears when one clicks on the project details button. See image below and kindly fix that. 

![project details](https://user-images.githubusercontent.com/65852621/225970629-7939f7c6-5050-4f63-ae29-22ae0431274b.PNG)